microsoftgraph/microsoft-graph-explorer
# add skipTest flag to sample queries

## Overview

Adds a skipTest flag to the sample queries that enables one to set skip the tests they do not want/need to run.

By default all tests are ran.

### Demo

N/A

### Notes

N/A

## Testing Instructions

* run this command: npm run import:samples
* check the gen-queries.ts file generated and set the skipTest flag of the first 5 tests to true
* Run this command: npm run test
* You should see that there are 245 tests executed. There are currently a total of 250 tests
